lib/em-hiredis/connection_manager.rb

```diff
@@ -154,8 +165,7 @@ def on_disconnected(prev_state)
       # state when we emit :disconnected and :reconnected, so we should only
       # proceed here if our state has not been touched.
       return unless @sm.state == :disconnected
-
-      if @reconnect_attempt > 3
+      if @reconnect_attempt > @options[:reconnect_attempts]
```

Comment:
I'm curious how are you expecting "infinite" to be expressed?

Reply:
the functionality right now does not support infinite reconnect. We decided with @leesio to do smth similiar to `ruby-rb` library to be consistent, and they don't support infinite reconnect. So I was going to pass smth like `reconnect_attempts = 32,767` for now

Reply:
That makes sense. Consistency is good :)